composeApp/build.gradle.kts

```diff
@@ -174,14 +174,15 @@ compose.desktop {
             packageVersion = "1.0.0"
             description = "YouTube Clone Using Kotlin Multiplatform"
             copyright = "© 2024 Muhammad Khubaib Imtiaz. All rights reserved."
+            val basePath = "processedResources/jvm/main/youtube_music"
             windows {
-                iconFile.set(project.file("composeApp/src/commonMain/composeResources/youtube_music.png"))
+                iconFile.set(layout.buildDirectory.file("$basePath.ico"))
             }
             macOS {
-                iconFile.set(project.file("composeApp/src/commonMain/composeResources/"))
+                iconFile.set(layout.buildDirectory.file("$basePath.icns"))
             }
             linux {
-                iconFile.set(project.file("composeApp/src/commonMain/composeResources/"))
+                iconFile.set(layout.buildDirectory.file("$basePath.png"))
             }
```

Comment:
On my system, I was not able to compile with the previous paths. Also .png icons supposedly only work on Windows, see https://github.com/JetBrains/compose-multiplatform/tree/master/tutorials/Native_distributions_and_local_execution#app-icon

Reply:
(needs the new icon files below as well though)

Reply:
Hey Stefan,

Kindly Enhancement The Readme file, let me add you as a contributor.